src/screens/SiteDetailScreen.js

```diff
@@ -13,7 +13,7 @@ import _ from 'lodash';
 
 import {attemptToAddFavorite, attemptToRemoveFavorite, attemptToUploadNewSite} from '../actions';
 
-import {linkColorBlue, navyBlueButton, hyperlinkBlue, campsiteIcon} from '../styles';
+import {linkColorBlue,gradientBlue,gradientMinte, navyBlueButton,mossGreen,limeGreenTitle,grayBlueDark,headerSemiWhiteTransparent,headerWhiteMediumTransparent, headerWhite, overlayBlue, headerWhiteTransparent, hyperlinkBlue, campsiteIcon, bloodOrange, sunsetOrange, boogerGreen,navBarBlue, eggShellWhite} from '../styles';
```

Comment:
The gradient looks awesome, but near the bottom of the page it's really hard to read the white text. Also, for things that don't take up the full width, you can see the background color stopping halfway. And for longer sitedetails, there seems to be some brown color that starts at the bottom of the page as well.